### PR TITLE
Html java script- Awesome books

### PR DESCRIPTION
**Pull request by mahdinoori2000**
### Added features
- Create a collection that keeps a list of books (hint: you can use an array of objects for that).
- Create a function to add a new book to the collection, with title and author.
- Create a function to remove a book from the collection by filter() method.
- Display all books saved in the collection at the top part of the page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,193 @@
...
+- **Dynamics of HTML using JavaScript**
+- **DOM Manipulation**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]()
```
Comment (wayungi) [inline README.md]: - [ ] Live demo link is broken. Kindly update the link or delete it if the project does not require deployment 👍🏾

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @mahdinoori2000 , 

image


**Congratulations on the good job done so far!** 🤝 

There are some issues that you still need to work on to go to the next project but you are almost there! 😄 

## Highlights ⚡ ⚡ 
- [ ] All linter checks are passing ✔️ 
- [ ] Fair contribution by both partners ✔️ 
- [ ]  Correct GitHub flow ✔️ 
- [ ] Descriptive PR body ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mahdinoori2000** [comment]
### What is newly added
- [ ] Add live demo link to the project.

**Review by LionRouge1 — APPROVED** [review]
Hi @mahdinoori2000, 

# STATUS: APPROVED! 🥇 💯 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

done

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**